src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeAction.cs

```diff
@@ -210,120 +193,10 @@ private IMethodSymbol CreateInequalityOperator(Compilation compilation, SyntaxGe
                     ImmutableArray.Create(generator.ReturnStatement(expression)));
             }
 
-            private async Task<IMethodSymbol> CreateGetHashCodeMethodAsync(CancellationToken cancellationToken)
```

Comment:
All this code was moved (untouched) into the GenerateEqualsAndGetHashCodeService.  That way we could reuse ths logic here and inside the "convert anonymous type to class" code.

Reply:
i could not find a way to do this (even putting the other class after this one) that made the diff look already.  sorry!